src/main/java/org/spongepowered/api/text/format/TextColors.java

```diff
@@ -46,6 +47,11 @@ private TextColors() {
         private final Color color = Color.BLACK;
 
         @Override
+        public CatalogKey getKey() {
+            return CatalogKey.sponge("none");
```

Comment:
We really should get Text rewrite done to eliminate this implementation leakage.......